From 1aa678f383102e4295cfe09611eb210840d45f6f Mon Sep 17 00:00:00 2001
From: Paul Barker <paul.barker@sancloud.com>
Date: Fri, 6 Aug 2021 09:59:07 +0100
Subject: [PATCH 2/2] ARM: dts: am335x-sancloud-bbe: Drop usb wifi comment

The wifi chip on USB port 4 may not be present on all BBE variants.

Signed-off-by: Paul Barker <paul.barker@sancloud.com>
---
 arch/arm/boot/dts/am335x-sancloud-bbe-common.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/am335x-sancloud-bbe-common.dtsi b/arch/arm/boot/dts/am335x-sancloud-bbe-common.dtsi
index 1d6fdd9de759..8fc869afbd25 100644
--- a/arch/arm/boot/dts/am335x-sancloud-bbe-common.dtsi
+++ b/arch/arm/boot/dts/am335x-sancloud-bbe-common.dtsi
@@ -93,6 +93,5 @@ usb2512b: usb-hub@2c {
 		compatible = "microchip,usb2512b";
 		reg = <0x2c>;
 		reset-gpios = <&gpio0 29 GPIO_ACTIVE_LOW>;
-		/* wifi on port 4 */
 	};
 };
-- 
2.30.2

